### PR TITLE
Additional tests of null PDBIds

### PR DESCRIPTION
**Pull request by sbliven**
This addresses #1019 and was developed in parallel with @josemduarte's fix #1020. It has been rebased to remove duplicated effort, and now consists mostly of additional tests for the bug.

### CODE REVIEW

**Review by aalhossary — APPROVED** [review]
Thanks! LGTM

**Review by aalhossary — APPROVED** [review]
Thanks! LGTM

File: biojava-structure/src/main/java/org/biojava/nbio/structure/io/PDBFileParser.java
Comment (josemduarte) [inline biojava-structure/src/main/java/org/biojava/nbio/structure/io/PDBFileParser.java:375]: It seems that `String.isBlank()` is a java 11 feature that we can't use just yet. At least that's what IntelliJ tells me. Interestingly the tests passed in CI, not sure why.

In any case, I'd like to move to java 11 soon. I think it is long overdue. We'll try to address that separately.

File: biojava-structure/src/main/java/org/biojava/nbio/structure/io/cif/CifStructureConsumerImpl.java
Comment (josemduarte) [inline biojava-structure/src/main/java/org/biojava/nbio/structure/io/cif/CifStructureConsumerImpl.java:899]: Same comment

File: biojava-structure/src/test/resources/AF-A0A0R4IYF1-F1-model_v2.pdb
Comment (josemduarte) [inline biojava-structure/src/test/resources/AF-A0A0R4IYF1-F1-model_v2.pdb]: Could you make this file smaller? Simply keep 2-3 residues from ATOM and SEQRES

**Review by josemduarte — CHANGES_REQUESTED** [review]
Thank you. Very needed tests!

**Review by josemduarte — APPROVED** [review]
I'll approve, `isBlank()` will be ok once #1053 is merged

**Comment by josemduarte** [comment]
I'll supersede this PR by a new one so that I can truncate the large PDB file before merging

**Pull request closed without merging**
